docs/how_to/install.rst

```diff
@@ -210,16 +210,15 @@ After each of the steps below run ``cms check`` to verify that you have resolved
 Sekizai
 =======
 
-`Django Sekizai <https://github.com/ojii/django-sekizai>`_ is required by the CMS for static files management. You need
-to have::
+`Django Sekizai <https://github.com/ojii/django-sekizai>`_ is required by the CMS for static files management. For that, you need to install the package, add it to installed apps and to your context_processors by following steps::
```

Comment:
Sekizai is one of the requirements of the cms so we shouldn't need to specify installing it separately.

See here: https://github.com/django-cms/django-cms/blob/develop/setup.py#L13

Reply:
Yes, I can remove the install line. I think for some reasons, it wasn’t installed for me and I had to manually install it and packaging library which was why I added it in the docs